<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeatherClass</class>
 <widget class="QWidget" name="WeatherClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Weather</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>256</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="filterButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>filter</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filterLineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="list">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>show list</string>
   </property>
  </widget>
  <widget class="QPushButton" name="computeTotalButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>compute total</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="totalLineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="weather.qrc"/>
 </resources>
 <connections/>
</ui>
